Dockerfile.agent

```diff
@@ -11,25 +11,27 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM centos:7 as build
+FROM ubuntu:20.04 as builder
 
-RUN yum -y update \
-	&& yum -y install fuse-devel pam-devel wget install gcc automake autoconf libtool make
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y python3 python3-jinja2 python3-setuptools meson wget gcc cmake libfuse2 libfuse-dev uuid-runtime systemd help2man pkg-config
 
-ENV LXCFS 4.0.9
+ENV LXCFS 5.0.2
 RUN wget https://linuxcontainers.org/downloads/lxcfs/lxcfs-$LXCFS.tar.gz && \
 	mkdir /lxcfs && tar xzvf lxcfs-$LXCFS.tar.gz -C /lxcfs  --strip-components=1 && \
-	cd /lxcfs && ./configure && make install
+	cd /lxcfs && \
+	meson setup -Dinit-script=systemd --prefix=/usr build/ && \
+	meson compile && \
+	meson install -C build
 
-FROM gcr.io/distroless/base@sha256:d8244d4756b5dc43f2c198bf4e37e6f8a017f13fdd7f6f64ec7ac7228d3b191e
+FROM ubuntu:20.04
```

Comment:
Do we need multi-stage builds if we always need ubuntu?

Reply:
Yes, it still reduce size of the agent image. I did try to use distroless but that didn't work smoothly.

Reply:
I missed a shared library. Updated and switched back to distroless image.